packages/asc-ui/src/components/FormTitle/FormTitle.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import FormTitle from './FormTitle'
+
+import Card from '../Card'
+import CardContent from '../Card/CardContent'
+import Header from '../Header'
+import Heading from '../Heading'
+import Paragraph from '../Paragraph'
+
+export default {
+  title: 'Experimental/Atoms/FormTitle',
+
+  decorators: [
+    (storyFn: () => React.ReactNode) => (
+      <div
+        style={{
+          padding: '0px 10px',
+        }}
+      >
+        {storyFn()}
+      </div>
+    ),
+  ],
+
+  parameters: {
+    notes: {
+      markdown:
+        '#### See component in <a href="https://designsystem.amsterdam.nl/7awj1hc9f/p/537bb7-title-component" target="_blank">Design system</a>',
```

Comment:
Can we use the same style here as in the other components? So a seperate README.md file with the component status as 'Experimental'. See: https://github.com/Amsterdam/amsterdam-styled-components/blob/master/packages/asc-ui/src/components/Accordion/README.md

Reply:
Sure!